.pre-commit-config.yaml

```diff
@@ -0,0 +1,16 @@
+files: ^js/app.js|css/styles.css|index.html # only website specific files
+repos:
+  - repo: https://github.com/pre-commit/pre-commit-hooks
+    rev: v3.4.0
+    hooks:
+      - id: check-yaml
+      - id: check-json
+      - id: check-merge-conflict
+      - id: check-case-conflict
+      - id: end-of-file-fixer
+      - id: mixed-line-ending
+        args: [--fix=lf]
+  - repo: https://github.com/pre-commit/mirrors-eslint
+    rev: ''  # Use the sha / tag you want to point at
+    hooks:
+      - id: eslint
```

Comment:
```suggestion
      - id: eslint
      
```

Reply:
I have added the file to the files checked by pre-commit, so it enforces this new line now.